packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -49,7 +49,7 @@ export default class LinkCore extends React.Component<Props> {
 
         const defaultStyles = [
             sharedStyles.shared,
-            !(hovered || focused || pressed) && restingStyles,
```

Comment:
tl;dr: we want the resting style to be part of the default style and add new state styles on top.

Context: The browser (chrome, firefox) automatically adds focus to the link if it's the active element when tabbing back to the page. This triggers ClickableBehavior's handleFocus(), which sets focused to false. So then the resting styles go away. This removes the underline for inline links since the default sharedStyles has textDecoration: "none". We can avoid this by just keeping the resting styles there by default and then adding new state styles on top of that.